### PR TITLE
updated patch handling for OSP16

### PR DESCRIPTION
**Pull request by gaelrehault**

### CODE REVIEW

File: src/pilot/install-director.sh [hunk 1]
```diff
@@ -272,69 +301,107 @@ echo "## Updating .bash_profile..."
...
+    echo
+    echo "## Patching Ironic iDRAC driver drac.py on ${container} ..."
+    upload_file_to_container "${container}" "${HOME}/pilot/drac.patch" "/tmp/drac.patch"
+    run_on_container "${container}" "patch -b -s /usr/lib/python3.6/site-packages/ironic/conf/drac.py /tmp/drac.patch"
+    run_on_container "${container}" "rm -f /usr/lib/python2.7/site-packages/ironic/conf/drac.pyc"
+    run_on_container "${container}" "rm -f /usr/lib/python2.7/site-packages/ironic/conf/drac.pyo"
+    echo "## Done"
+  done
+
+# Update Drac patches
```
Comment (heydog15) [inline src/pilot/install-director.sh:344]: Should this be "Update iDRAC patches"?

Comment (gaelrehault) [reply src/pilot/install-director.sh:344]: refers to the dracclient libraries ( python-dracclient )

File: src/pilot/install-director.sh [hunk 2]
```diff
@@ -272,69 +301,107 @@ echo "## Updating .bash_profile..."
...
+# 2. sync_power_state_interval is increased to work around an issue where
+#    servers go into maintenance mode in ironic if polled for power state too
+#    aggressively.
 echo
-echo "## Restarting httpd"
-sudo systemctl restart httpd
+echo "## Patching ironic.conf..."
+apply_patch "sudo patch -b -s /var/lib/config-data/puppet-generated/ironic_api/etc/ironic/ironic.conf ${HOME}/pilot/ironic.patch"
+apply_patch "sudo patch -b -s /var/lib/config-data/puppet-generated//ironic/etc/ironic/ironic.conf ${HOME}/pilot/ironic.patch"
+echo "## Done."
```
Comment (dpaterson) [inline src/pilot/install-director.sh:381]: nit "generated//ironic" should be "generated/ironic"

**Review by dpaterson — APPROVED** [review]

**Comment by heydog15** [comment]
Merging so the Ironic team can look at why it is failing.

**Pull request merged**